### ISSUE TITLE
When running code in Jupyter NoteBook

### ISSUE THREAD

**Issue by AlirezaBoroon** [microsoft/vscode#137174]
Issue Type: <b>Bug</b>

Hi
I have a problem with running program in Jupyter NoteBook, when running, sometimes the VS Code doesn't show the printed stuff at the moment, and if I go forward and one step forward, it will show the previous printed line, and I can't see that line, at that correct moment, and I have to go and one step forward in the program running to see the details of the previous step.
Therefore please solve this issue.
Thanks.

VS Code version: Code 1.62.2 (3a6960b964327f0e3882ce18fcebd07ed191b316, 2021-11-11T20:56:38.428Z)
OS version: Windows_NT x64 10.0.19043
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz (8 x 1800)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|6.18GB (1.45GB free)|
|Process Argv|--crash-reporter-id 27f72fee-8da4-4129-bb28-02f0269ce2ab|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (8)</summary>

Extension|Author (truncated)|Version
---|---|---
theme-monokai-pro-vscode|mon|1.1.19
python|ms-|2021.11.1422169775
vscode-pylance|ms-|2021.11.1
jupyter|ms-|2021.10.1101450599
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.3
polacode|pnp|0.3.4
material-theme|zhu|3.13.2

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonvspyt551cf:30345471
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30384385
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
vsccppwtct:30382698
bridge0708:30335490
dockerwalkthru:30377721
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsqsis200:30386379
vsaa593:30376534
vssld246cf:30386378

```

</details>

**Comment by rebornix** [comment]
@AlirezaBoroon can you please share a recording of what you were seeing to help troubleshoot?

**Comment by AlirezaBoroon** [comment]
[https://mega.nz/file/2uB0SRQA#QiRSf96dmpa4A0Dx1YfqlPBAJig5SgSJlTxzRVcnYSE](url)
Here I recorded the screen, while running the program.
About the program: this program removes the letter entered by the user, like what I did (the letter "k") in a string.
As we see, I tried to enter the letter k, two times, second one was for a conditional question, and was not shown at a correct moment.
I hope it will help.
